src/config/dynamic-theme-fixes.config

```diff
@@ -604,6 +604,16 @@ INVERT
 
 ================================
 
+alt.hololive.tv
+
+INVERT
```

Comment:
Very loosy selectors. Unless your absolute sere that it won't mess up other things, try to "hardened" it a bit.